src/components/utdragFraSykefravaeret/UtdragFraSykefravaeret.tsx

```diff
@@ -148,7 +147,7 @@ export const Sykmeldinger = ({
   );
   return (
     <div className="utdragFraSykefravaeret__sykmeldinger">
-      <Undertittel>{tekster.sykmeldinger.header}</Undertittel>
+      <Undertittel tag={"h3"}>{tekster.sykmeldinger.header}</Undertittel>
```

Comment:
Dette er for å ha mest mulig semantisk riktig HTML, da disse lå under overskriften som var en h2.